### ISSUE TITLE
Bunch of errors in 0.0.7

### ISSUE THREAD

**Issue by eugenet8k** [humbertogontijo/homeassistant-roborock#57]
```
Unexpected exception importing platform custom_components.roborock.vacuum
Traceback (most recent call last):
  File "/Users/pi/.homeassistant-venv/lib/python3.9/site-packages/homeassistant/loader.py", line 728, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/Users/pi/.homeassistant-venv/lib/python3.9/site-packages/homeassistant/loader.py", line 745, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/Cellar/python@3.9/3.9.15/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/pi/.homeassistant/custom_components/roborock/vacuum.py", line 255, in <module>
    class RoborockVacuum(RoborockCoordinatedEntity, StateVacuumEntity, ABC):
  File "/Users/pi/.homeassistant/custom_components/roborock/vacuum.py", line 366, in RoborockVacuum
    def capability_attributes(self) -> Mapping[str, Any] | None:
TypeError: unsupported operand type(s) for |: '_GenericAlias' and 'NoneType'
```

```
Unable to prepare setup for platform roborock.vacuum: Platform not found (Exception importing custom_components.roborock.vacuum).
```

```
Platform roborock does not generate unique IDs. ID _ogzWh1iWAK_n6kTEBFPog already exists - ignoring camera.upstairs_vacuum_2
```

```
Platform roborock does not generate unique IDs. ID is_water_box_carriage_attached__ogzWh1iWAK_n6kTEBFPog already exists - ignoring binary_sensor.upstairs_vacuum_mop_attached
Platform roborock does not generate unique IDs. ID is_water_box_attached__ogzWh1iWAK_n6kTEBFPog already exists - ignoring binary_sensor.upstairs_vacuum_water_box_attached
```

**Comment by humbertogontijo** [comment]
I've removed the or(|) from the typing and fixed the generated unique_ids in 0.0.8. The problem is that everyone might need to reinstall the integration 😪

**Comment by Trebacz** [comment]
I just updated to 0.0.8 code - removed and readded the integration. I did not uninstall through HACS. Seems to be back to normal.

**Comment by eugenet8k** [comment]
Thank you @humbertogontijo for the quick fix, everything is back to normal. Also, I see more diagnostic sensors, which is awesome. Thank you for all your contributions.

**Comment by mvdwetering** [comment]
> I've removed the or(|) from the typing and fixed the generated unique_ids in 0.0.8. The problem is that everyone might need to reinstall the integration 😪

@humbertogontijo FYI you can add `from __future__ import annotations` at the top of the file (before any other imports I think) to use | annotations and other newer annotation features on older Python versions.
